assets/bot_commands/de.md

```diff
@@ -1,4 +1,4 @@
-Änderungssatz `{ch_id}`
+Änderungssatz [{ch_id}]({osm_link}) - „{comment}“
```

Comment:
Is this kind of double quote a German thing or just a typo?

Reply:
German double quotes are at the bottom on start and top at the end. See also https://en.wikipedia.org/wiki/Quotation_mark#Summary_table